doc/whats_new.rst

```diff
@@ -12,6 +12,56 @@ Version 0.18
 Changelog
 ---------
 
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **The ``model_selection`` module**
+
+    The classes and functions from the formerly :mod:`sklearn.cross_validation`,
+    :mod:`sklearn.grid_search` and :mod:`sklearn.learning_curve` are
+    grouped together into a single module available at
+    :mod:`sklearn.model_selection`. There are some major changes to these
+    newly moved classes and functions. Refer below to know about the
+    changes.
```

Comment:
I wonder if we should make this sound less like "moving old tools, with some breaking changes" and more like "we have a new module that allows a much smoother workflow, including nested cross-validation and better manipulation of parameter searches with Pandas; many things will stay the same but there are some key differences, read on for details".

The users who don't follow the master branch (most users, I assume?) will probably learn about these changes from here, so we want to be clear that they're well motivated and better for the future as well.